services/static-webserver/client/source/class/osparc/navigation/Manuals.js

```diff
@@ -139,10 +139,16 @@ qx.Class.define("osparc.navigation.Manuals", {
         menu.add(newFogbugzIssueBtn);
       }
 
-      if ((statics.s4lSupportEmail || statics.s4lliteSupportEmail) && (osparc.utils.Utils.isProduct("s4l") || osparc.utils.Utils.isProduct("s4llite"))) {
-        const giveFeedbackBtn = new qx.ui.menu.Button(qx.locale.Manager.tr("Give us Feedback"));
-        giveFeedbackBtn.addListener("execute", () => this.__openSendEmailFeedbackDialog(statics), this);
-        menu.add(giveFeedbackBtn);
+      if (osparc.utils.Utils.isProduct("s4l") || osparc.utils.Utils.isProduct("s4llite")) {
+        const forumBtn = new qx.ui.menu.Button(qx.locale.Manager.tr("S4L Forum"));
+        forumBtn.addListener("execute", () => window.open("https://forum.zmt.swiss/"), this);
```

Comment:
we could add a new link in the product's table for this as well

Reply:
![image](https://user-images.githubusercontent.com/33152403/200840398-47e86917-6174-467d-b704-4432c7509cab.png)